bigtable/hbase/snippets/src/main/java/com/example/bigtable/Reads.java

```diff
@@ -237,6 +236,38 @@ public static void readPrefix(String projectId, String instanceId, String tableI
   }
   // [END bigtable_reads_prefix_hbase]
 
+  // [START bigtable_reverse_scan_hbase]
+  public static void readRowsReversed() {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String instanceId = "my-instance-id";
+    String tableId = "mobile-time-series";
+    readRowsReversed(projectId, instanceId, tableId);
+  }
+
+  public static void readRowsReversed(String projectId, String instanceId, String tableId) {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (Connection connection = BigtableConfiguration.connect(projectId, instanceId)) {
+      Table table = connection.getTable(TableName.valueOf(tableId));
+      Scan revScan =
+          new Scan()
+              .setReversed(true)
+              .setLimit(2)
+              .withStartRow(Bytes.toBytes("phone#4c410523#20190505"));
+      ResultScanner rows = table.getScanner(revScan);
+
+      for (Result row : rows) {
+        printRow(row);
+      }
+    } catch (IOException e) {
```

Comment:
can you get rid of the try catch and just let the method throw the exception 

https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md#exception-handling

Reply:
I think that should come separately, to keep the structure consistent with the rest of the file.

Reply:
– will refactor in a larger PR

Reply:
+1 to that. If we are not demonstrating what to do with the exception, it should be thrown.